packages/components/src/components/SquigglePlayground.tsx

```diff
@@ -420,10 +304,137 @@ export const SquigglePlayground: FC<PlaygroundProps> = ({
       </HeadedSection>
     </div>
   );
+};
+
+const PlayControls: React.FC<{
+  autoplay: boolean;
+  playing: boolean;
+  stale: boolean;
+  onAutoplayChange: (value: boolean) => void;
+  play: () => void;
```

Comment:
the above variable names could be improved. Use ``isX`` for booleans, and I'd suggest having more descriptive names. This function isn't very long, so longer names won't be that bad.